scripts/exregional_bias_correction_o3.sh

```diff
@@ -0,0 +1,461 @@
+  cp_vrfy -rL "${AQM_AIRNOW_HIST_DIR}/bcdata.${yyyymm}/airnow" "${DATA}/data/bcdata.${yyyymm}"
+  cp_vrfy -rL "${AQM_AIRNOW_HIST_DIR}/bcdata.${yyyymm}/interpolated" "${DATA}/data/bcdata.${yyyymm}" 
+fi
+
+# Retrieve real-time airnow data for the last three days
+if [ "${DO_REAL_TIME}" = "TRUE" ]; then
+  mkdir -p ${DATA}/data/bcdata.${yyyymm_m1}/airnow/${yyyy_m1}/${PDYm1}/b008
+  mkdir -p ${DATA}/data/bcdata.${yyyymm_m2}/airnow/${yyyy_m2}/${PDYm2}/b008
+  mkdir -p ${DATA}/data/bcdata.${yyyymm_m3}/airnow/${yyyy_m3}/${PDYm3}/b008
+
+  cp_vrfy ${COMINairnow}/${yyyy_m1}/${PDYm1}/b008/xx021 ${DATA}/data/bcdata.${yyyymm_m1}/airnow/${yyyy_m1}/${PDYm1}/b008
+  cp_vrfy ${COMINairnow}/${yyyy_m2}/${PDYm2}/b008/xx021 ${DATA}/data/bcdata.${yyyymm_m2}/airnow/${yyyy_m2}/${PDYm2}/b008
+  cp_vrfy ${COMINairnow}/${yyyy_m3}/${PDYm3}/b008/xx021 ${DATA}/data/bcdata.${yyyymm_m3}/airnow/${yyyy_m3}/${PDYm3}/b008
+fi
+
+#-----------------------------------------------------------------------------
+# STEP 2:  Extracting PM2.5, O3, and met variables from CMAQ input and outputs
+#-----------------------------------------------------------------------------
+
+case $cyc in
+  00) bc_interp_hr=06;;
+  06) bc_interp_hr=72;;
+  12) bc_interp_hr=72;;
+  18) bc_interp_hr=06;;
+esac
+
+ic=1
+while [ $ic -lt 120 ]; do
+  if [ -s ${COMIN}/${NET}.${cycle}.chem_sfc.f0${bc_interp_hr}.nc ]; then
+    echo "cycle ${cyc} post1 is done!"
+    break
+  else
+    (( ic=ic+1 ))
+  fi
+done
+
+if [ $ic -ge 120 ]; then
+  print_err_msg_exit "FATAL ERROR - COULD NOT LOCATE:${NET}.${cycle}.chem_sfc.f0${bc_interp_hr}.nc"
+fi
+
+# remove any pre-exit ${NET}.${cycle}.chem_sfc/met_sfc.nc for 2-stage post processing
+DATA_grid="${DATA}/data/bcdata.${yyyymm}/grid"
+if [ -d "${DATA_grid}/${cyc}z/${PDY}" ]; then
+  rm_vrfy -rf "${DATA_grid}/${cyc}z/${PDY}"
+fi
+
+mkdir_vrfy -p "${DATA_grid}/${cyc}z/${PDY}"
+ln_vrfy -sf ${COMIN}/${NET}.${cycle}.chem_sfc.*.nc ${DATA_grid}/${cyc}z/${PDY}
+ln_vrfy -sf ${COMIN}/${NET}.${cycle}.met_sfc.*.nc ${DATA_grid}/${cyc}z/${PDY}
+
+#-----------------------------------------------------------------------------
+# STEP 3:  Intepolating CMAQ O3 into AIRNow sites
+#-----------------------------------------------------------------------------
+
+mkdir_vrfy -p ${DATA}/data/coords 
+mkdir_vrfy -p ${DATA}/data/site-lists.interp 
+mkdir_vrfy -p ${DATA}/out/ozone/${yyyy}
+mkdir_vrfy -p ${DATA}/data/bcdata.${yyyymm}/interpolated/ozone/${yyyy} 
+
+cp_vrfy ${PARMaqm_utils}/bias_correction/sites.valid.ozone.20220724.12z.list ${DATA}/data/site-lists.interp
+cp_vrfy ${PARMaqm_utils}/bias_correction/aqm.t12z.chem_sfc.f000.nc ${DATA}/data/coords
+cp_vrfy ${PARMaqm_utils}/bias_correction/config.interp.ozone.7-vars_${id_domain}.${cyc}z ${DATA}
+
+PREP_STEP
+${EXECdir}/aqm_bias_interpolate config.interp.ozone.7-vars_${id_domain}.${cyc}z ${cyc}z ${PDY} ${PDY} || print_err_msg_exit "Call to executable to run AQM_BIAS_INTERPOLATE returned with nonzero exit code."
```

Comment:
Should this use `eval $RUN_CMD_SERIAL` that was defined above along with redirecting output/error to pgmout/pgmerr?